examples/02_decoding/plot_haxby_searchlight.py

```diff
@@ -13,6 +13,7 @@
 # Load Haxby dataset
 # -------------------
 import numpy as np
```

Comment:
Can you please import numpy at relevant section/cell ? Makes it easier in user guide